packages/buttons/src/views/Anchor.js

```diff
@@ -34,30 +36,51 @@ const StyledAnchor = styled.a.attrs({
   ${props => retrieveTheme(COMPONENT_ID, props)};
 `;
 
+const StyledNewWindowIcon = styled(NewWindowIcon)`
+  width: 0.75em;
+  height: 0.75em;
+  vertical-align: middle;
+
+  ${props => (isRtl(props) ? `margin-right: ${zdSpacingXxs};` : `margin-left: ${zdSpacingXxs};`)}
+`;
+
 /**
  * Accepts all `<a>` props
  */
-const Anchor = ({ focused, ...other }) => (
-  <KeyboardFocusContainer>
-    {({ getFocusProps, keyboardFocused }) => (
-      <StyledAnchor
-        {...getFocusProps({
-          ...other,
-          focused: focused || keyboardFocused
-        })}
-      />
-    )}
-  </KeyboardFocusContainer>
-);
+const Anchor = props => {
+  // eslint-disable-next-line
+  const { focused, external, children, theme, ...other } = props;
+  const rtl = isRtl(props);
+
+  return (
+    <KeyboardFocusContainer>
+      {({ getFocusProps, keyboardFocused }) => (
+        <StyledAnchor
+          {...getFocusProps({
+            ...other,
+            focused: focused || keyboardFocused
+          })}
+        >
+          {rtl && external && <StyledNewWindowIcon />}
+          {children}
+          {!rtl && external && <StyledNewWindowIcon />}
```

Comment:
If the page has `direction` set then that should handle the correct position of the icon right?

Reply:
From the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/direction)

> The direction CSS property sets the direction of text, table columns, and horizontal overflow

Testing locally it didn't affect the ordering of the SVG

Reply:
Yeah fair enough not sure what lead me to believe that.